cli/admin/src/test/java/io/pravega/cli/admin/AbstractTlsAdminCommandTest.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.cli.admin;
+
+import io.pravega.client.ClientConfig;
+import io.pravega.client.stream.impl.DefaultCredentials;
+import io.pravega.local.LocalPravegaEmulator;
+import io.pravega.test.common.SecurityConfigDefaults;
+import io.pravega.test.common.TestUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.Timeout;
+
+import java.net.URI;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class AbstractTlsAdminCommandTest {
+
+    // Security related flags and instantiate local pravega server.
+    private static final Integer CONTROLLER_PORT = TestUtils.getAvailableListenPort();
```

Comment:
Do not make these static. Keeping state (available ports is a form of state) in static variables prevents these tests from being run in parallel.

Reply:
Made these non static.